BUILD.bazel

```diff
@@ -1,6 +1,6 @@
 # Copyright 2019 OpenAPI-Generator-Bazel Contributors
 
-load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+load("@rules_pkg//:pkg.bzl", "pkg_tar", "pkg_deb")
```

Comment:
@wing328 pkg_tar is deprecated (extracted) from bazel, have to import seperately: https://docs.bazel.build/versions/2.2.0/be/pkg.html

How it imported: https://github.com/OpenAPITools/openapi-generator-bazel/pull/41/files#diff-5493ff8e9397811510e780de47c57abb70137f1afe85d1519130dc3679d60ce5